### DISCUSSION TITLE
Make minecart follow rails in angles and ramps ?

### DISCUSSION THREAD

**Discussion by craftish37** [PurpurMC/Purpur#524] [__Dismissed__] [answered]
After overkilling minecart speed (need to be a great way to move on the server), I see a problem with this speed in angles : minecarts completely acts crazy when going in angles with a too great speed (linked 1). Moreover, according to the reality, this rails disposition taken by minecarts at great speed must to bump minecarts out of the rails, make it fly a short time (or long), and go on following rails who are farer. It could be good for any server who wants to make minecart really useful and not just a way to make attractions or quick hopper lines.

linked 1: https://youtu.be/wyKqOcrIX88
linked 2: 
<img width="849" alt="Capture d’écran 2021-07-27 à 17 19 44" src="https://user-images.githubusercontent.com/63811617/127182308-d247c742-afb8-41f7-942c-87adc53e4f17.png">

P.S.: When testing linked 2, I had issues with minecarts who goes backward when hitting the ramp

**Comment by BillyGalbreath** [comment, marked as answer]
This is an unfortunate side effect that has plagued all plugins and mods that increase the speed of minecarts. You'll have to deal with it like everyone else.

In fact, I didn't want to add this config option to purpur at all and explained that this is exactly why. But people were insistent and someone else PRed it, so whatever.
